packages/Vendor/src/ValueObjects/VendorApprovalRecord.php

```
@@ -0,0 +1,47 @@
+    private string $approvedByUserId;
+
+    private \DateTimeImmutable $approvedAt;
+
+    private ?string $approvalNote;
+
+    public function __construct(
+        string $approvedByUserId,
+        \DateTimeImmutable $approvedAt,
+        ?string $approvalNote = null,
+    ) {
+        $normalizedApprovedByUserId = trim($approvedByUserId);
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $normalizedApprovedByUserId. Keep variable name length under 20. (undefined)

(LongVariable)